auth.js

```diff
@@ -0,0 +1,45 @@
+const bcrypt = require('bcryptjs');
+const LocalStrategy = require('passport-local').Strategy;
+
+const knex = require('./connectDB');
```

Comment:
If you are making an import of connectDB a good variable name would be **databaseConnection**